deploy/kubernetes/overlays/dev/master_image.yaml

```diff
@@ -8,4 +8,4 @@ spec:
     spec:
       containers:
         - name: efs-plugin
-          image: amazon/aws-efs-csi-driver:latest
+          image: amazon/aws-efs-csi-driver:master
```

Comment:
We might need to add ImagePullPolicy: Always since only kubelet only always pull the ":latest" tag  without a policy. 

> If you would like to always force a pull, you can do one of the following:

> set the imagePullPolicy of the container to Always.
omit the imagePullPolicy and use :latest as the tag for the image to use.
omit the imagePullPolicy and the tag for the image to use.
enable the AlwaysPullImages admission controller.


https://kubernetes.io/docs/concepts/containers/images/#updating-images

Reply:
good point, fixed